docs/installation_guides/validation_of_installation.md

```diff
@@ -0,0 +1,14 @@
+# Validate installation 
```

Comment:
Should this be a separate document? in the 3scale_guides.md this information is listed as a section. Should this be added to other documents?

This is a general question and I am looking for your opinion.

Reply:
To me, validation of installation should be a separate page since we have split the installation with OLM vs Local, so it makes me think it's logical to have the verification as separate step. 
An alternative to this is to duplicate the verification and list it both in olm and local, however, this brings duplication which I think we can avoid.

Reply:
Ya I am happy with that. I will say the more I look at the command for checking if the installation is complete, the more I think that could be convert into a one liner for all types. That update is outside of this work.